packages/frontend-sfe/src/pages/api/newsletter/[...newsletter].ts

```diff
@@ -0,0 +1,77 @@
+import { env } from '@lib/environment'
```

Comment:
This file has a funky file name 😂

Reply:
Haha true, I'm already quite used to this Next.js syntax. It just means to really catch all api-routes nested under `api/newsletter`: https://nextjs.org/docs/api-routes/dynamic-api-routes#catch-all-api-routes (but not necessary here, as I'm not going deeper than level 1. but still, does not hurt.)